src/utils/image.js

```diff
@@ -219,30 +106,28 @@ export async function deleteImage(hash) {
 	const settings = await getSettings();
 	const host = await getHostSettings();
 
-	if (host === aHosts[1]) {
-		await doRequest(
-			hash,
-			'GET',
-			null,
-			null,
-			() => {},
-			() => {},
-			() => {}
-		);
-	} else {
-		await doRequest(
-			`https://api.imgur.com/3/image/${hash}`,
-			'DELETE',
-			null,
-			{
-				text: 'Authorization',
-				value: `Client-ID ${settings.apiClientId}`
-			},
-			() => {},
-			() => {},
-			() => {}
-		);
-	}
+	const get = host === aHosts[1];
```

Comment:
NotLikeThis 
hosts.js should export an object with named keys instead of an array, not clear what's going on here if it stays like that

Reply:
Can you show an example?
I'm not sure what you mean by that.

It's currently an array so I can easily iterate over it to show the several hosts.

Reply:
arrays are just fancy objects in js anyway, but essentially what we want here is to know what is what and where and why and how, so it should look somewhat like this:

```js
export default {
     namedKey: {
         key: property,
         key: property
     },
     ...
}
```

so that you can then go ahead and access them
```js
import thing from 'file';

switch (something):
     case: (thing.namedKey):
           break;
     ...
```